semantic_conventions/logs/media.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: log
```

Comment:
```suggestion
  - id: attributes.log
```

Reply:
Addressed in [f93d5c3](https://github.com/open-telemetry/semantic-conventions/pull/133/commits/f93d5c35763e3d5636860fc25404de8f4f532e43).